ts/IterableInboxClickableRow.tsx

```diff
@@ -1,38 +1,38 @@
 'use strict'
 
 import React, { useState } from 'react'
+import { StyleSheet, TouchableOpacity } from 'react-native'
+
 import IterableInboxMessageListItem from './IterableInboxMessageListItem'
-import {
-   StyleSheet,
-   TouchableOpacity
-} from 'react-native'
 import Message from './messageType'
 
 type MessageClickableRowProps = {
    index: number,
    message: Message,
    handleMessageSelect: Function,
-   last: boolean 
 }
 
-const IterableInboxClickableRow = ({ index, message, handleMessageSelect, last }: MessageClickableRowProps) => {
+const IterableInboxClickableRow = ({ 
+   index, 
```

Comment:
do you even still need this `index` prop passed down here? from what i can tell, neither this component, nor `IterableInboxMessageListItem` where it is passed down further make any use of it.

Reply:
The index prop is not needed. I will make the necessary update.